sklearn/utils/sparsefuncs_fast.pyx

```diff
@@ -278,7 +278,10 @@ def inplace_csr_row_normalize_l1(X):
     _inplace_csr_row_normalize_l1(X.data, X.shape, X.indices, X.indptr)
 
 
-def _inplace_csr_row_normalize_l1(np.ndarray[floating, ndim=1] X_data, shape,
+@cython.boundscheck(False)
+@cython.wraparound(False)
+@cython.cdivision(True)
+cdef _inplace_csr_row_normalize_l1(np.ndarray[floating, ndim=1] X_data, shape,
                                   np.ndarray[int, ndim=1] X_indices,
```

Comment:
You've one character above but not a space here to match the indentation. Is this still PEP8 compliant?